src/App.jsx

```diff
@@ -1,14 +1,32 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList';
+
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+
+function getUserById(userId) {
+  return usersFromServer.find(user => userId === user.id) || null;
+}
+
+function getCommentById(id) {
+  return commentsFromServer.filter(comments => id === comments.postId) || null;
```

Comment:
The function `getCommentById` uses `filter`, which returns an array. If you intend to get a single comment by its ID, consider using `find` instead of `filter` to return a single object or `null` if not found.